flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/ddl/SqlAlterTableReset.java

```diff
@@ -33,13 +33,16 @@
 
 import static java.util.Objects.requireNonNull;
 
-/** ALTER TABLE [[catalogName.] dataBasesName].tableName RESET ( 'key1' [, 'key2']*). */
+/** ALTER TABLE IF EXISTS [[catalogName.] dataBasesName].tableName RESET ( 'key1' [, 'key2']*). */
 public class SqlAlterTableReset extends SqlAlterTable {
     private final SqlNodeList propertyKeyList;
 
     public SqlAlterTableReset(
-            SqlParserPos pos, SqlIdentifier tableName, SqlNodeList propertyKeyList) {
-        super(pos, tableName, null);
+            SqlParserPos pos,
```

Comment:
Maybe change to `SqlAlterTableReset(SqlParserPos, SqlIdentifier, SqlNodeList, boolean)`?